CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+<!--changelog-start-->
```

Comment:
Do you really need these tags? I think the plugin would import the entire file by default.

Reply:
In a previous version the import would fail without these annotations. include-markdown "../CHANGELOG.md" just fails